src/node.rs

```diff
@@ -244,6 +244,18 @@ impl<S: StorageExt> SawtoothRaftNode<S> {
 
         let is_leader = self.raw_node.raft.state == raft::StateRole::Leader;
         if is_leader {
+            //check if we stepped up as leader from follower state
```

Comment:
Minor issue, but please keep comment formatting consistent (space after `//`, start with capital letter)

Reply:
Updated the commit